xml/System/Half.xml

```diff
@@ -1104,14 +1180,21 @@
         <Parameter Name="result" Type="System.Half" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="s">The input to be parsed.</param>
-        <param name="style">One of the enumeration values that specifies how to parse the input.</param>
-        <param name="provider">A format provider.</param>
-        <param name="result">When this method returns, a value that represents the input span. If the input exceeds the range of <see cref="T:System.Half" />, its value is <see cref="P:System.Half.PositiveInfinity" /> or <see cref="P:System.Half.NegativeInfinity" />. If the parse was unsuccessful, contains the default <see cref="T:System.Half" /> value.</param>
-        <summary>Tries to parse a span with the specified <see cref="T:System.Globalization.NumberStyles" /> and <see cref="T:System.IFormatProvider" />.</summary>
+        <param name="s">A read-only span that contains a number to convert.</param>
+        <param name="style">A bitwise combination of enumeration values that indicates the permitted format of <paramref name="s"/>.</param>
+        <param name="provider">An object that supplies culture-specific formatting information about <paramref name="s"/>.</param>
+        <param name="result">When this method returns, contains Half-precision floating-point number equivalent to the numeric value or symbol contained in <paramref name="s"/>, if the conversion succeeded, or a default <see cref="T:System.Half"/> value if the conversion failed. The conversion fails if the <paramref name="s"/> parameter is <see langword="null"/> or <see cref="F:System.String.Empty"/> or is not a number in a valid format. If is a valid number less than <see cref="P:System.Half.MinValue"/>, result is <see cref="P:System.Half.NegativeInfinity"/>. If is a valid number greater than <see cref="P:System.Half.MaxValue"/>, result is <see cref="PositiveInfinity"/>. This parameter is passed uninitialized; any value originally supplied in result will be overwritten.</param>
```

Comment:
This one's causing the latest build warning:

```suggestion
        <param name="result">When this method returns, contains Half-precision floating-point number equivalent to the numeric value or symbol contained in <paramref name="s"/>, if the conversion succeeded, or a default <see cref="T:System.Half"/> value if the conversion failed. The conversion fails if the <paramref name="s"/> parameter is <see langword="null"/> or <see cref="F:System.String.Empty"/> or is not a number in a valid format. If is a valid number less than <see cref="P:System.Half.MinValue"/>, result is <see cref="P:System.Half.NegativeInfinity"/>. If is a valid number greater than <see cref="P:System.Half.MaxValue"/>, result is <see cref="P:System.Half.PositiveInfinity"/>. This parameter is passed uninitialized; any value originally supplied in result will be overwritten.</param>
```

Reply:
Found some other errors too. Fixed them!